xtask/src/ci.rs

```diff
@@ -69,6 +69,9 @@ pub fn ci(args: CiJob, metadata: CargoMetadata) -> cross::Result<()> {
             if target.has_ci_image() {
                 gha_output("has-image", "true")
             }
+            if target.is_default_test_image() {
```

Comment:
I've added this so when `cross` and `zig` images are added, we can just exclude them with a single `matches!` statement later.

Reply:
I've also implemented the same change mentioned [here](https://github.com/cross-rs/cross/pull/880#pullrequestreview-1025458032) for the zig image.